bowtie/_benchmarks.py

```diff
@@ -436,6 +511,33 @@ def serializable(self):
         ]
         return as_dict
 
+    @classmethod
+    def from_dict(
+        cls,
+        data: dict[str, Any],
+    ) -> BenchmarkReport:
+        return cls(
+            metadata=BenchmarkMetadata.from_dict(data["metadata"]),
+            results={
+                r.get("name"): BenchmarkGroupResult.from_dict(r)
```

Comment:
Why would `name` not be present? `None` is going to be a really poor key for this dict.

Reply:
`name` would be present always. I just used get here. I'll change it to r["name"].